git_pile/git_pile.py

```diff
@@ -666,7 +666,7 @@ def gen_cover_letter(diff, output, n_patches, baseline, pile_commit, prefix, ran
 def gen_full_tree_patch(output, n_patches, oldbaseline, newbaseline, oldref, newref, prefix, add_header):
     # possibly too big diff, just avoid it for now
     if oldbaseline != newbaseline:
-        return
```

Comment:
Not sure but doesn't the above line return sort of an empty object, whereas returning None returns a non-empty object? So I was thinking maybe the only change needed was "if tail: print(tail)" below. But you probably tested this and this works :-)

Reply:
```
>>> def bla():
...     return
... 
>>> def bla2():
...     return None
... 
>>> def bla3():
...     pass
... 
>>> a = bla()
>>> b = bla2()
>>> c = bla3()
>>> a is None
True
>>> b is None
True
>>> c is None
True
```

They are the same thing.  But as a C programmer I find it confusing when we mix the return styles.

Reply:
Thanks for the explanation 👍